### DISCUSSION TITLE
TPA Message

### DISCUSSION THREAD

**Discussion by NecroSeigen** [Zrips/CMI#6783] [Feature Requests]
When using the `/tpa` command, you can click directly in the chat to accept the request. As shown below: 
![image](https://user-images.githubusercontent.com/81390182/161031743-f11c4927-f60d-4d27-9ef2-532d937cedbd.png)

However, we cannot change the command that is executed. In this case it is `/cmi tpaccept` but when hiding commands from the players, for example with PL-HIDE-PRO, one is obliged to leave `/cmi tpaccept`, so that the player can accept. It should therefore be possible to change the command that is executed, for example, instead of `/cmi tpaccept`, put `/tpaccept`.

**Comment by barannnnnns** [comment]
You can execute the command from console.

**Reply by barannnnnns** [reply]
this way you can hide the plugin.

**Reply by NecroSeigen** [reply]
How can I do it with the console?

**Comment by barannnnnns** [comment]
Read the sections at the top of the lang file.


# Full color code support and some variables

# Keep in mind that variables wont work for some lines, when it will for anothers :)

# Just keep them where there are now and everything will be ok :)

# Some lines can have global variables set. For player who will be effected. In example /heal Zrips then Zrips data will be used

# [serverName] to show server name

# [playerName] to show target player name

# [playerDisplayName] to show target player display name

# [lvl] to show target player level

# [exp] to show target player total exp

# [hp] to show target player health

# [maxHp] to show target player max health

# [hunger] to show target player hunger level

# [gameMode] to show target player gamemode

# [prefix] to show target player prefix if possible

# [suffix] to show target player suffix if possible

......

# To have more than one JSON message use <Next>

# <C> performs command as a player who clicked

# <CC> performs command from console once

# <CCI> performs command from console every time player clicks text

# <URL> includes url


.....

**Comment by barannnnnns** [comment]
ssss# To have more than one JSON message use <Next>

sss# <C> performs command as a player who clicked

sss# <CC> performs command from console once

sss# <CCI> performs command from console every time player clicks text

sss# <URL> includes url

**Comment by barannnnnns** [comment]
Unfortunately, the part I threw is not visible, but you can do this by reading the upper part of your language file.

**Comment by MrCrazys** [comment]
> When using the `/tpa` command, you can click directly in the chat to accept the request. As shown below: ![image](https://user-images.githubusercontent.com/81390182/161031743-f11c4927-f60d-4d27-9ef2-532d937cedbd.png)
> 
> However, we cannot change the command that is executed. In this case it is `/cmi tpaccept` but when hiding commands from the players, for example with PL-HIDE-PRO, one is obliged to leave `/cmi tpaccept`, so that the player can accept. It should therefore be possible to change the command that is executed, for example, instead of `/cmi tpaccept`, put `/tpaccept`.

Well, from what I've seen, it wouldn't exactly make a difference because doing `/tpaccept` executes `/cmi tpaccept` anyways. Unless I've misunderstood you.
